docs/04-operation-guides/operations/obsv-00-enable-configurable-logging.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Enable the telemetry component (beta feature)
+---
+
+## Context
+
+<!-- Briefly provide background information for the task so that the users understand the purpose of the task and what they will gain by completing the task correctly. This section should be brief and does not replace or recreate a concept topic on the same subject, although the context section might include some conceptual information.
+-->
+
+With Kyma's telemetry component, you can provide your custom configuration for Fluent Bit, for example, to integrate with vendors like [VMWare](https://medium.com/@shrishs/log-forwarding-from-fluent-bit-to-vrealizeloginsightcloud-9eeb14b40276) using generic outputs, or with any vendor via a [fluentd integration](https://medium.com/hepsiburadatech/fluent-logging-architecture-fluent-bit-fluentd-elasticsearch-ca4a898e28aa) using the forward output. This enables you to integrate Kyma's logging backend with your own observability systems outside the Kyma cluster.
+
+The new telemetry component ships a new Fluent Bit installation alongside with the new telemetry operator, which will support the dynamic configuration of the Fluent Bit. By default, it will not ship any logs.
+The new `global.telemetry.enabled` flag on the logging component disables the existing Fluent Bit installation. Furthermore, it activates a new log pipeline resource to enable log shipment to the existing Loki stack.
```

Comment:
```suggestion
The new `global.telemetry.enabled` flag on the logging component disables the existing Fluent Bit installation. Furthermore, it activates a new `LogPipeline` resource to enable log shipment to the existing Loki stack.
```
It is the name of our new resource type

Reply:
However, we're trying to avoid CamelCase and [use natural language](https://github.com/kyma-project/kyma/issues/12473) in docs.

Reply:
Then use at least Log Pipeline

Reply:
```suggestion
The new `global.telemetry.enabled` flag on the logging component disables the existing Fluent Bit installation. Furthermore, it activates a new Log Pipeline resource to enable log shipment to the existing Loki stack.
```